### PR TITLE
installation: usage of Flask-Security-Fork

### PR DESCRIPTION
**Pull request by jirikuncar**
Signed-off-by: Jiri Kuncar jiri.kuncar@cern.ch

### CODE REVIEW

**Comment by lnielsen** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -64,6 +67,10 @@ before_install:
   - "requirements-builder --level=min setup.py > .travis-lowest-requirements.txt"
   - "requirements-builder --level=pypi setup.py > .travis-release-requirements.txt"
   - "requirements-builder --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "if [ ! -f /home/krnowak/cernbox/geckodriver-v0.11.1-linux32.tar.gz ]; then wget https://github.com/mozilla/geckodriver/releases/download/v0.11.1/geckodriver-v0.11.1-linux32.tar.gz -P $HOME/selenium; tar xvfz $HOME/selenium/geckodriver-v0.11.1-linux32.tar.gz -C $HOME/selenium; fi"
```
Comment (jirikuncar) [inline .travis.yml]:
```diff
- ! -f /home/krnowak/cernbox/geckodriver-v0.11.1-linux32.tar.gz
+ ! -f $HOME/selenium/geckodriver-v0.11.1-linux32.tar.gz
```

**Comment by krzysztof** [comment]
@jirikuncar I'm removing the tar.gz in before_install command, as this is specific to installation, and not a periodic tmp files that are created inside the cache folder (which seems to be the main purpose of "before_cache".

File: .travis.yml [hunk 2]
```diff
@@ -64,6 +67,10 @@ before_install:
   - "requirements-builder --level=min setup.py > .travis-lowest-requirements.txt"
   - "requirements-builder --level=pypi setup.py > .travis-release-requirements.txt"
   - "requirements-builder --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "if [ ! -f $HOME/selenium/geckodriver ]; then wget https://github.com/mozilla/geckodriver/releases/download/v0.11.1/geckodriver-v0.11.1-linux32.tar.gz -P $HOME/selenium; tar xvfz $HOME/selenium/geckodriver-v0.11.1-linux32.tar.gz -C $HOME/selenium; rm -f $HOME/selenium/geckodriver-v0.11.1-linux32.tar.gz; fi"
+  - "mkdir /home/travis/.local/bin -p"
+  - "PATH=$PATH:/home/travis.local/bin"
```
Comment (jirikuncar) [inline .travis.yml]: `$HOME/selenium`

**Pull request merged**